src/Homepage/Form/Form.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react';
+
+import './_form.scss';
+
+import InputLocation from "../Location/InputLocation";
+import CustomDropdown from "../Collections/CustomDropdown";
+import SearchButton from "../Button/SearchButton";
+
+const API_Collections = "https://api.unsplash.com/collections/featured/?client_id=930640e0b7713dca3ab1a0751b6f4b4741d1dfca6a72be2a071cddd6c1d0c92c";
```

Comment:
Storing API base URL and credentials in a separate file would make more sense. And also your naming convention should be convenient. So `API_Collections` should be `API_COLLECTIONS`.

Reply:
I solved in Listpage.